Leave/leave_interface.py

```diff
@@ -52,25 +52,28 @@ async def HandleLeaveReactions(client, payload):
     message = await channel.fetch_message(payload.message_id)
     embed = message.embeds[0]
     
-    Action = UI.ParseEmoji(payload.emoji)
-    if Action == None:
+    action = UI.ParseEmoji(payload.emoji)
+    if action == None:
         return
 
-    if not (utils.isNotBot(payload.member) and utils.IsAdmin(payload.member)):
+    if not (utils.isNotBot(payload.member)):
```

Comment:
defeats the purpose of a helper function. just use the `bot` attribute on discord.user

Reply:
100% correct, and also will be more readable